### DISCUSSION TITLE
스키마 적다가 isValidObjectId 보고 생각났는데요

### DISCUSSION THREAD

**Discussion by ghost** [tastekim/WeAllLie-BE#19] [Ideas]
그러고 보니까 그때 몽구스에서는 _id를 바로 사용할 수 있도록 변환해줬던거 같았어요..
이게 되면 바로 따로 primal Key 역할의 field를 만들 필요 없이 이걸로 이용하면 될 것도 같아요...; ㅁ;!!

<img width="730" alt="image" src="https://user-images.githubusercontent.com/112525911/201255072-605c3782-ba72-40df-9851-cdbd8bf330e6.png">
<img width="718" alt="image" src="https://user-images.githubusercontent.com/112525911/201255101-651461f5-6dbe-4576-8bde-010c9a92095d.png">

**Comment by tastekim** [comment]
,,,,?! 
그렇다면 그 자동생성되는 id 필드이름을 저희가 수정할 수 있는지도 알아봐야겠네요 !
